C/plugins/common/include/http_sender.h

```diff
@@ -56,6 +56,25 @@ class HttpSender
 		virtual void setOCSClientSecret      (std::string& OCSClientSecret) = 0;
 		virtual void setOCSToken             (std::string& OCSToken) = 0;
 
+        /**
+         * @brief Constructs the file path for the OMF log based on environment variables.
+         *
+         * This function checks for the existence of two environment variables:
```

Comment:
The implementation of _getOMFTracePath()_ now calls _getDataDir()_ which is great. Because of that change, this comment is no longer accurate. The function no longer requires the existence of _FLEDGE_ROOT_ and _FLEDGE_DATA_ and will no longer return an empty string if neither is set.

Reply:
ok